posthog/hogql_queries/insights/funnels/base.py

```diff
@@ -315,6 +315,25 @@ def _serialize_step(
     def extra_event_fields_and_properties(self):
         return self._extra_event_fields + self._extra_event_properties
 
+    @property
+    def _absolute_actors_step(self) -> Optional[int]:
+        """The actor query's 1-indexed target step converted to our 0-indexed SQL form. Never a negative integer."""
+        if self.context.actorsQuery is None:
+            return None
+        target_step = self.context.actorsQuery.funnelStep
+        if target_step is None:
+            return None
```

Comment:
```suggestion
        if self.context.actorsQuery is None or self.context.actorsQuery.funnelStep is None:
            return None

        target_step = self.context.actorsQuery.funnelStep
```
nittiest of nit-picks, but somehow I find this more readable